internal/controlplane/handlers_profile.go

```diff
@@ -430,85 +442,120 @@ func (s *Server) GetProfileStatusByName(ctx context.Context,
 		dbEntity = &db.NullEntities{Entities: entities.EntityTypeToDB(e.GetType()), Valid: true}
 	}
 
-	if len(in.GetRule()) > 0 {
-		rule = &sql.NullString{String: in.GetRule(), Valid: true}
-	} else {
-		rule = &sql.NullString{Valid: false}
+	ruleType = &sql.NullString{
+		String: in.GetRuleType(),
+		Valid:  in.GetRuleType() != "",
+	}
+
+	// TODO: Remove deprecated 'rule' field from proto
+	if !ruleType.Valid {
+		//nolint:staticcheck // ignore SA1019: Deprecated field supported for backward compatibility
+		ruleType = &sql.NullString{
+			String: in.GetRule(),
+			Valid:  in.GetRule() != "",
```

Comment:
Non-blocking, but one of `rule_type` or `rule` should be set, right?  (Otherwise, this is very neat logic, very nice!)

Reply:
No, `rule` or `rule_type` and `rule_name` both are optional. They are associated with `--rule/--ruleType` and `--ruleName` flags respectively, so if user doesn't specify one, we return the complete eval status for a policy.